examples/select_items/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+
+	clarify "github.com/clarify/clarify-go"
+	"github.com/clarify/clarify-go/query"
+)
+
+func main() {
+	creds, err := clarify.CredentialsFromFile("clarify-credentials.json")
+	if err != nil {
+		panic(err)
+	}
+
+	ctx := context.Background()
+	client := creds.Client(ctx)
+
+	// To select or publish signals, you must enable access to the "admin"
+	// namespace in clarify; this allows selecting signals from all integrations
+	// in the organization, but for the example, we will use the integration
+	// from the credentials file.
+	result, err := client.SelectItems().Filter(query.Comparisons{
```

Comment:
I get `Method not found` for `clarify.SelectItems` request

Reply:
Which credentials are you using? Should work against dev (clarifyapp.clarify.cloud) and (clarifyapp.clarify.io) instances, but won't work against the scratch1 instance, unless you have rebased on master.

Reply:
I've tested on `dev` and all methods work fine except of `SelectItems`, but I'll check it again

Reply:
Tested on `dev` and `prod` and now it works fine. Probably forgot to update credentials for this case

Reply:
Thanks for re-testing 🙇